### ISSUE TITLE
In Chrome, search begins but goes back to default Google search when a key is pressed

### ISSUE THREAD

**Issue by allenap** [huhu/rust-search-extension#101]
- I type _rs_ and space/tab into the address bar.
- "Rust Search Extension | " appears.
- I start typing, but it immediately goes back to the default Google search.

![2021-01-30 15 54 49](https://user-images.githubusercontent.com/348564/106359539-c1b00980-6313-11eb-8d51-7a7e9a158d1b.gif)

It was working fine the last time I used it ~2 weeks ago.
I've tried restarting Chrome, disabling and reenabling the extension.

Google Chrome 88.0.4324.96 (Official Build) (x86_64)
macOS Catalina 10.15.7 (19H114)

**Comment by Folyd** [comment]
Hi @allenap. Thanks for the report. This is weird. I can't reproduce this on my machine (same environment).  Do you have tried to reinstall the extension?

**Comment by allenap** [comment]
Thanks @Folyd. Reinstalling (with a restart of Chrome in the middle) hasn't helped. I have also tried disabling all of my other plugins, and removing a few that were old and I wasn't using, and restarting Chrome. Is there something else I can do to debug what might be causing this?

**Comment by Folyd** [comment]
Sure, you can inspect the extension by this way:
1. Right-click the extension's icon, choosing `Manage extensions`
![image](https://user-images.githubusercontent.com/3369694/106380471-6fdbb200-63ed-11eb-95a7-ce53c4ec1a3d.png)
2. Click the `background page` to open the inspect window.
3. Check the `console` tab. 

If you can see any error logs, that would be helpful.

**Comment by allenap** [comment]
Sadly, no logs at all.

I tried the extension in a new Chrome profile and it worked fine. Back in the profile where it's broken, I tried clearing out local storage for the plugin, then restarting Chrome, but it didn't fix the issue.

If I think of anything else I'll comment.

**Comment by Folyd** [comment]
Then I think this has nothing to do with the extension. May you have a try with [https://cpp.extension.sh](https://cpp.extension.sh)? I guess the same problem will occur.

**Comment by allenap** [comment]
> May you have a try with https://cpp.extension.sh?

That one works correctly. This is a weird puzzle.

**Comment by allenap** [comment]
I figured it out: there was a search engine registered (in "Manage search engines") with the keyword _rs_:

![image](https://user-images.githubusercontent.com/348564/106387632-eb803380-63da-11eb-9ecd-ddbbeef64097.png)

Removing that fixed the issue.

I don't remember when I configured it, but I know it's old, and certainly from well before the last time I used rust-search-extension. Perhaps the recent release of Chrome 88 changed the way keywords are resolved in the address bar. In any case, Chrome certainly appears to have a bug in how it handles keywords.

Thanks for your help @Folyd.
